presto-main/src/main/java/com/facebook/presto/operator/JoinHashSupplier.java

```diff
@@ -68,7 +70,6 @@ public JoinHashSupplier(
         else {
             positionLinksFactoryBuilder = ArrayPositionLinks.builder(addresses.size());
         }
-
```

Comment:
Please revert it (or introduce as a separate commit).

Reply:
Revert the deletion of a line?